3-node-cluster/ansible/playbooks/setup-pgpool.yml

```diff
@@ -54,6 +62,15 @@
           backend_data_directory2 = '{{ PG_CIRRUS_INSTALLATION_DIRECTORY }}' # Data directory path for standby2 PostgreSQL server.
         insertafter: EOF
 
+    # Create a pgpool password file in /etc/pgpool2
```

Comment:
Are we setting the proper permissions for this file as well? i.e, 
Allowing only the postgres user for read/write operations

Reply:
We are using this task to write to pool_passwd file which I believe gets created when pgpool is installed
I think I should update the commend and name of the task

    # Create a pgpool password file in /etc/pgpool2
    - name: Create a pool_passwd file
      lineinfile:
        path: /etc/pgpool2/pool_passwd
        line: |
          repuser:{{ REPUSER_PASSWORD }}
          postgres:{{ PFILE_PASSWORD }}
        insertafter: EOF

Reply:
The permissions of pool_passwd file are as following:

-rw-r----- 1 root postgres   30 Aug  1 12:08 pool_passwd